conf/docker-dev-env/solr/conf/schema.xml

```diff
@@ -0,0 +1,1554 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Does this mean we have to maintain two copies of schema.xml and solrconfig.xml? I hope not! 😅